sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -1008,27 +1008,6 @@ def DeletePermission(
         permission_id = base.GetResourceIdOrFullNameFromLink(permission_link)
         return self.DeleteResource(path, "permissions", permission_id, None, options, **kwargs)
 
-    def ReadItems(
```

Comment:
I wouldn't remove this. While I agree that it's largely redundant, and was never intended to be public - unfortunately it _did_ leak out into the public API. So unless there's a really strong reason to remove, I would just leave it there.

Reply:
This applies to the other removed methods as well.

Reply:
rip, missed the fact that since we call it in the public method it is technically public surface, will add it back!

Reply:
the other removed methods were actually just leftover though - they truly had no uses in the SDK/ were never called by any other methods in the public surface area

Reply:
My point being is this whole internal connection object _was_ public surface area - albeit accidentally.

Reply:
I thought the underscore and the fact we don't export it made it private though? why are you saying this is public?

Reply:
re-added them